src/Compilers/Core/Portable/PortableShim.cs

```diff
@@ -174,6 +186,11 @@ internal static class File
                 .GetDeclaredMethod(nameof(Delete), new[] { typeof(string) })
                 .CreateDelegate<Action<string>>();
 
+            internal static readonly Action<string, string> Move = Type
```

Comment:
This API is not available in Net Core 4.5.  It only became available in Net Core 5.0.  Do we ship in Net Core 4.5 environments?  If so this can't be used.